dbt_sugar/core/task/doc.py

```diff
@@ -118,8 +117,7 @@ def move_name_and_description_to_first_position(self, model: Dict[str, Any]):
         """
         # DEPRECATION: Drop ordered dict when dropping python 3.6 support
         ordered_dict = OrderedDict(model)
-        if ordered_dict.get("description"):
-            ordered_dict.move_to_end("description", last=False)
```

Comment:
that is going to be needed I had to add that in the bootstrap task. Did you have an issue with it?